source/features/default-branch-button.css

```diff
@@ -0,0 +1,12 @@
+.rgh-highlight-non-default-branch {
+	background-color: var(
+		--bgColor-accent-muted,
+		var(--color-accent-subtle)
```

Comment:
I think these colors have been removed or deprecated years ago, we don’t need the fallbacks. We should probably remove them from the whole repo (if I’m right) and replace them with `red` so that these bugs are immediately seen, reported and fixed.

Reply:
I mean in this PR let’s only change this file. The rest of the repo can follow separately

Reply:
<img width="1588" alt="image" src="https://github.com/user-attachments/assets/af2f1b6b-1e1d-439f-b341-52030fab8ab3">